velox/docs/functions/spark/math.rst

```diff
@@ -266,6 +266,16 @@ Mathematical Functions
 
     Returns the secant of ``x``.
 
+.. function:: sign(x) -> double
+
+    Returns the signum function of ``x``. It returns:
```

Comment:
We can copy the presto function document
```
.. function:: sign(x) -> [same as x]

    Returns the signum function of ``x``. For both integer and floating point arguments, it returns:
    * 0 if the argument is 0,
    * 1 if the argument is greater than 0,
    * -1 if the argument is less than 0.

    For double arguments, the function additionally return:
    * NaN if the argument is NaN,
    * 1 if the argument is +Infinity,
    * -1 if the argument is -Infinity.
```

Reply:
We can't use this directly, in Spark this function only supports double and internal types.

Reply:
In that case, we should add the Supported types for ``x`` are Double